server/test/articlePagination.spec.js

```diff
@@ -0,0 +1,58 @@
+import chai, { expect } from 'chai';
```

Comment:
We don't need a new file to test this. Go to the article.spec.js file and update the get all articles route test